libs/components/core/src/lib/modules/media-query/provide-media-query-service-override.ts

```diff
@@ -0,0 +1,25 @@
+import { Provider, Type, inject } from '@angular/core';
+
+import { SKY_MEDIA_QUERY_SERVICE_OVERRIDE } from './media-query-service-override.token';
+import { SkyMediaQueryService } from './media-query.service';
+
+/**
+ * Overrides the `SkyMediaQueryService` provider, usually within an element injector.
+ * @internal
+ */
+export function provideSkyMediaQueryServiceOverride(
+  svc: Type<SkyMediaQueryService>,
```

Comment:
Should this be `Type<SkyMediaQueryServiceInterface>`?

Reply:
I could be misunderstanding, but the `Type` interface needs to reference a "class" type. If I change it to the interface, I get a TypeScript error, `Type 'SkyMediaQueryServiceInterface' is missing the following properties from type 'SkyMediaQueryService': #currentSubject, #breakpointChange, #breakpointChangeObs, #currentBreakpoint, and 7 more.ts(2740)` because it's expecting the token value to be an instance of `SkyMediaQueryService`. Is there something I'm missing in my assumptions?